src/index.html

```diff
@@ -7,12 +7,76 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__logo">
+        <img
+          class="logo"
+          src="images/imac.jpeg"
+          alt="imac"
+        />
+      </div>
+
+      <div class="info">
+        <h2 class="info__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="info__product-code">Product code: 195434</p>
+      </div>
+
+      <div class="reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="reviews__text">Reviews: 5</p>
+      </div>
+
+      <div class="cost">
+        <p class="cost__title">Price:</p>
+
+        <p class="cost__sum">$2,199</p>
+      </div>
+
+      <button
+        class="buy-now"
+        data-qa="hover"
+      >
```

Comment:
Fix it everywhere if you have only two attributes
```suggestion
      <button class="card__button" data-qa="hover">
```